packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.cpp

```diff
@@ -16,15 +16,14 @@ namespace winrt::SampleAppCpp::implementation {
 /// WinMain().
 /// </summary>
 App::App() noexcept {
-  MainComponentName(L"SampleApp");
-  JavaScriptMainModuleName(L"index.windows");
+  InstanceSettings().MainComponentName(L"SampleApp");
```

Comment:
Why does this need to change? I think if you're using the easy-mode "ReactApplicaton", we should maintain these properties as pass-throughs to the properties on the host.

Doin this should also fix the CI build errors you're seeing.

Reply:
The idea was to simplify theReactApplication API. I have restored these properties for now.

---
In reply to: [374356328](https://github.com/microsoft/react-native-windows/pull/4009#discussion_r374356328) [](ancestors = 374356328)